tool/pylib/generator/output/Script.py

```diff
@@ -128,7 +128,10 @@ def createResourceStruct(resources, formatAsTree=False, updateOnlyExistingSprite
                 for glyphname, code in res.mapping.iteritems():
                   fdsc = "@%s/%s" % (res.alias, glyphname)
                   if not fdsc in result:
-                      result[fdsc] = [result[resid][1], round(result[resid][2] / code[1]), code[0]]
+                      try:
+                          result[fdsc] = [result[resid][1], round(result[resid][2] / code[1]), code[0]]
+                      except:
+                          pass
```

Comment:
There is no possibility that another problem could manifest on this line, rendering valid glyphs seemingly invalid ? Any it might take a long time do discover the reason ?